### PR TITLE
fix(editor): Usage and plans page on Desktop

### PR DESCRIPTION
**Pull request by cstuncsik**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/src/views/SettingsUsageAndPlan.vue [hunk 1]
```diff
@@ -53,34 +53,36 @@ const onLicenseActivation = async () => {
 };
 
 onMounted(async () => {
-	usageStore.setLoading(true);
-	if (route.query.key) {
+	if (!usageStore.isDesktop) {
```
Comment (ivov) [inline packages/editor-ui/src/views/SettingsUsageAndPlan.vue:56]: Nit - inverting the check and returning early would prevent the nesting and diff.

File: packages/editor-ui/src/views/SettingsUsageAndPlan.vue [hunk 2]
```diff
@@ -110,99 +112,122 @@ const onDialogClosed = () => {
 const onDialogOpened = () => {
 	activationKeyInput.value?.focus();
 };
+
+const openPricingPage = () => {
+	sendUsageTelemetry('desktop_view_plans');
+	window.open('https://n8n.io/pricing', '_blank');
```
Comment (ivov) [inline packages/editor-ui/src/views/SettingsUsageAndPlan.vue:118]: Ideally the URL in a constant.

**Review by ivov — APPROVED** [review]
LGTM. Only minor comments so feel free to merge.

**Comment by cstuncsik** [comment]
Thanks @ivov , agree but I have to leave now. If you have time now please update otherwise I'll just merge

**Comment by janober** [comment]
Got released with `n8n@0.209.4`

**Pull request merged**